packages/core/src/crypto/JwsService.ts

```diff
@@ -128,10 +137,10 @@ export class JwsService {
       }
     }
 
-    return { isValid: true, signerKeys: signerKeys }
+    return { isValid: true, signerKeys }
   }
 
-  private buildProtected(options: ProtectedHeaderOptions) {
+  private buildProtected(options: JwsProtectedHeaderOptions) {
     if (!options.jwk && !options.kid) {
       throw new AriesFrameworkError('Both JWK and kid are undefined. Please provide one or the other.')
     }
```

Comment:
Is it fine if they are both supplied?

Reply:
No. this is handled in a different place. i'll see to refactor, as it's a bit weird. But i did it like this for a specific reason

Reply:
Ah it seems they're already next to each other:

```
    if (!options.jwk && !options.kid) {
      throw new AriesFrameworkError('Both JWK and kid are undefined. Please provide one or the other.')
    }
    if (options.jwk && options.kid) {
      throw new AriesFrameworkError('Both JWK and kid are provided. Please only provide one of the two.')
    }
```